src/main/c/ssl.c

```diff
@@ -1616,7 +1617,7 @@ TCN_IMPLEMENT_CALL(jlong, SSL, getTime)(TCN_STDARGS, jlong ssl)
 
     UNREFERENCED(o);
 
-    return SSL_get_time(ssl_);
+    return SSL_get_time(ssl_->session);
```

Comment:
thanks @nmittler! see https://github.com/netty/netty-tcnative/issues/69.  This is used for every single `OpenSslSession` in Netty ... https://github.com/netty/netty/blob/master/handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java#L1298